### ISSUE TITLE
implement the prominent style for labelLarge and labelMedium in the TextTheme - Material Design 3

### ISSUE THREAD

**Issue by alessandrovolpi-puntozero** [flutter/flutter#147775]
### Use case

As described in the [documentation](https://m3.material.io/styles/typography/type-scale-tokens), in Material Design 3 there are three Label styles: 
- Label Large
- Label Medium
- Label Small.

For the Large and the Medium, Material Design provides two different type of weightness: normal and prominent.

But in the text_theme.dart there are only labelLarge and labelMedium, without the prominent style.

### Proposal

Implement the prominent style in the TextTheme for the labelLarge and labelMedium.

**Comment by danagbemava-nc** [comment]
Hi @alessandrovolpi-puntozero, it seems like on the material site, the weight for prominent is 700, does using fontWeight: FontWeight.w700 not suite your use-case?

**Comment by alessandrovolpi-puntozero** [comment]
Yes, but I think it’s better to have this style directly in the TextScheme instead of manual coding every times. 

For example, when I customize the textScheme I write:
```
textTheme = TextTheme(
  labelLarge: const TextStyle(fontSize: 16.0, fontWeight: FontWeight.w400),
  labelSmall: const TextStyle(fontSize: 12.0, fontWeight: FontWeight.w400)
);
```
My proposal is something like this:
```
textTheme = TextTheme(
  labelLarge: const TextStyle(fontSize: 16.0, fontWeight: FontWeight.w400),
  labelLargeProminent: const TextStyle(fontSize: 16.0, fontWeight: FontWeight.w700),
  labelSmall: const TextStyle(fontSize: 12.0, fontWeight: FontWeight.w400)
);
```

In this way when I want to use this style I can use textScheme.labelLargeProminent
